x/mongo/driver/auth/oidc.go

```diff
@@ -26,14 +26,24 @@ import (
 // MongoDBOIDC is the string constant for the MONGODB-OIDC authentication mechanism.
```

Comment:
These are now public so they can be reused in configuration error checking to keep consistency if we ever need to change the values.